docs/system/setup.rst

```diff
@@ -0,0 +1,66 @@
+============
+System Setup
+============
+
+
+Linux Machine Setup
+===================
+The target linux machine will need to have installed:
+
+* ansible
+* docker
+* nvidia-docker2
+* docker-compose
+
+You will want to add your user to the ``docker`` group to facilitate using
+docker commands.
+
+The network interface to which we are going to connect to the HoloLens2 should
+be appropriately configured:
+
+* IPv4 should be set to 192.168.1.100 (address), 255.255.255.0 (netmask), and
+  192.168.1.1 (gateway)
```

Comment:
This is repeated down below in "HL2 First-time Setup"

Reply:
This is slightly different: this describes using the host static IP to end if **`100`** while the HL2 section below describes the static IP to be **`101`**.